### ISSUE TITLE
Missing all delete options

### ISSUE THREAD

**Issue by schneckenschnabel** [multiSnow/mcomix3#115]
Neither in the normal view or in the library view is there any option for deleting an archive.
Where those simply removed (when yes, oh god why?) or is there simply a problem on my end?

Tried the fedora 32 build, and the git build from here, all the same.

**Comment by multiSnow** [comment]
Since https://github.com/multiSnow/mcomix3/commit/bcd3f3dbc41ca5cd6ea5b4894851179719e1e481, mcomix itself does not provide a way to delete something from disk.
Please do it through a file manager or some command like `rm` at your own risk.

**Comment by wyatt8740** [comment]
Boooo.

I used this feature a lot, and I don't want to give up an 'open with' option for the hotkey.

There was even a nice warning to make sure you wanted to delete something. I really don't know why this was removed completely. The way I see it, this is preventing feature-parity with the 2.x versions.

While it admittedly may be confusing for many cases, there are also use cases (like mine; I don't use archives or dual pages) where it makes perfect sense. I do understand the rationale, I just do not like it.
